rabbitmq/src/main/java/io/micronaut/rabbitmq/intercept/RabbitMQConsumerAdvice.java

```diff
@@ -80,203 +81,150 @@
     private final RabbitMessageSerDesRegistry serDesRegistry;
     private final ConversionService<?> conversionService;
     private final Map<String, ChannelPool> channelPools;
-    private final Map<Channel, ConsumerState> consumerChannels = new ConcurrentHashMap<>();
+    private final List<RecoverableConsumerWrapper> consumers = new CopyOnWriteArrayList<>();
 
     /**
      * Default constructor.
      *
-     * @param beanContext       The bean context
-     * @param binderRegistry    The registry to bind arguments to the method
-     * @param exceptionHandler  The exception handler to use if the consumer isn't a handler
-     * @param serDesRegistry    The serialization/deserialization registry
+     * @param beanContext The bean context
+     * @param binderRegistry The registry to bind arguments to the method
+     * @param exceptionHandler The exception handler to use if the consumer isn't a handler
+     * @param serDesRegistry The serialization/deserialization registry
      * @param conversionService The service to convert consume argument values
-     * @param channelPools      The channel pools to retrieve channels
+     * @param channelPools The channel pools to retrieve channels
      */
     public RabbitMQConsumerAdvice(BeanContext beanContext,
-                                  RabbitBinderRegistry binderRegistry,
-                                  RabbitListenerExceptionHandler exceptionHandler,
-                                  RabbitMessageSerDesRegistry serDesRegistry,
-                                  ConversionService<?> conversionService,
-                                  List<ChannelPool> channelPools) {
+            RabbitBinderRegistry binderRegistry,
+            RabbitListenerExceptionHandler exceptionHandler,
+            RabbitMessageSerDesRegistry serDesRegistry,
+            ConversionService<?> conversionService,
+            List<ChannelPool> channelPools) {
         this.beanContext = beanContext;
         this.binderRegistry = binderRegistry;
         this.exceptionHandler = exceptionHandler;
         this.serDesRegistry = serDesRegistry;
         this.conversionService = conversionService;
         this.channelPools = new HashMap<>(channelPools.size());
-        for (ChannelPool cp: channelPools) {
+        for (ChannelPool cp : channelPools) {
```

Comment:
Revert whitespace changes

Reply:
I did restore as much of the file layout to minimize the diff. But as i did some method extraction, there are still some pseudo changes.

I think it improves the readability, as the `process` method is now really focused on extracting the bean and method definitions and the invocation. The Rabbit client handling is now clearly separated.